docs/conventions.rst

```diff
@@ -68,10 +68,10 @@ and KDE Discover). The `Freedesktop AppStream specification
 <https://www.freedesktop.org/software/appstream/docs/>`_ provides a complete
 reference for providing AppData.
 
-AppData files should be named with the application ID and the ``.appdata.xml``
+AppData files should be named with the application ID and the ``.metainfo.xml``
 file extension, and should be placed in ``/app/share/metainfo/``. For example::
 
-  /app/share/metainfo/org.gnome.Dictionary.appdata.xml
+  /app/share/metainfo/org.gnome.Dictionary.metainfo.xml
```

Comment:
Before seeing this, I just submitted #331 which is not as complete, but then it mention that other legacy locations are accepted. I think it's worth mentioning.